src/store/modules/SplashModule.ts

```diff
@@ -41,6 +63,9 @@ export const SplashModule = {
 
             return item;
         },
+        updateRequestItem({ commit }, body: UpdateRequestItemBody) {
```

Comment:
Is this wrapper actually needed? Why not call the commit directly?

Reply:
Umm, why not call the commit from `Splash.vue`? Has there been some change to this on Vuex upgrade?

Reply:
Huh that's fair, will change